### DISCUSSION TITLE
Can we add rocketChat RN in our exist RN Project ?

### DISCUSSION THREAD

**Discussion by gulsher7** [RocketChat/Rocket.Chat.ReactNative#3512] [General]
hey everyone i have a question regarding react native..

can we add rocketchat RN in our exist RN project. i know this is silly question sorry for that. Pease help if anyone is know how to do that.

**Comment by diegolmello** [comment]
Hey, @gulsher7.
In theory you can, but I've never done that so I can't help you much.
